qa/tasks/rbd_fio.py

```diff
@@ -127,7 +127,7 @@ def run_fio(remote, config, rbd_test_dir):
 
     formats=[1,2]
     features=[['layering'],['striping'],['exclusive-lock','object-map']]
```

Comment:
@harishmunjulur I think we can fix the features here based on distro kernel testing(krbd) rather than modify the client config, I am closing this PR for now.

Reply:
Sure Thanks @vasukulkarni